interfaces/cython/cantera/test/test_reactor.py

```diff
@@ -437,6 +437,8 @@ def test_heat_flux_func(self):
         self.assertNear(U2a + Q, U2b, 1e-6)
 
     def test_mass_flow_controller(self):
+        import os
+        self.assertEqual(os.environ["OPENBLAS_NUM_THREADS"], "1")
```

Comment:
I’m still at a loss why this fails (albeit only by a narrow margin) when multi-threading is enabled. As mentioned earlier, I was under the impression that multithreading works reliably out of the box. Regardless, requiring that multi-threading is disabled here feels like a nuclear option to me …

Reply:
This was a temporary debugging change, I don't intend that this should be kept permanently. It's actually only a problem with multithreading when using 0.3.15 of OpenBLAS, which indicates (as @speth mentioned) a change two layers removed from us, and which changes a regression test (not an invariant like h = u + pv) close to the order of machine precision. I'm not particularly concerned about the failure...

Reply:
Sure - didn’t realize that this was still WIP (draft/no review status wasn’t set). Please disregard.

Reply:
Yeah, sorry about the WIP, I didn't realize I would have to debug this, I figured it'd be the same effect as in #1035. Ah well.

Reply:
Would have thought the same thing.